docs/includes/_gas_and_fees.md

```diff
@@ -0,0 +1,19 @@
+# Gas and Fees
+
+Gas represents the computational effor required to execute an operation. The fee is how much crypto to pay to execute a transaction. Insufficient gas will result in a failed transaction, and consumed gas.
```

Comment:
```suggestion
Gas represents the computational effor required to execute an operation. The fee is how much `INJ` to pay to execute a transaction. Insufficient gas will result in a failed transaction, and consumed gas.
```

Reply:
A failed transaction consumes gas?

Reply:
fixed. tried locally, and it does not. inj amount in portfolio was the same before and after.